### ISSUE TITLE
VSCode always asks for API key after starting the docker container

### ISSUE THREAD

**Issue by yudi-azvd** [wakatime/vscode-wakatime#315]
I run

    docker compose up -d

I open vscode and it asks me for the key:

![image](https://user-images.githubusercontent.com/37981839/205516719-fec9eb66-c1a4-417a-abbd-a095e989c5fb.png)

Is there a way to save it on my machine so that the wakatime extension can read it from inside even for a remote workspace? Or any other way.

By remote workspace I mean I'm using VSCode connected to a docker container using the [Dev Containers](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) extension. This container runs in my local machine

**Comment by yudi-azvd** [comment]
Yes it is related. I was searching the issues but I couldnt find. Thats why I opened this issue

I tried the suggestion in https://github.com/wakatime/vscode-wakatime/issues/180#issuecomment-832075272 but it didn't work:

```ini
[settings]
api_key = ...
standalone = false
alpha = true

# I've also tried like this:
standalone = false
alpha = true

[settings]
api_key = ...
```

None of this worked.

Should i move this to #180 and close this issue? Also, is there any logs I can provide?

**Comment by alanhamlett** [comment]
The `standalone` config option was removed in [17.0.0](https://github.com/wakatime/vscode-wakatime/blob/master/CHANGELOG.md#1700-2021-08-30) and currently `alpha` is the same as stable because wakatime-cli has [no pending alpha releases](https://github.com/wakatime/wakatime-cli/releases), so you can remove both those config lines.

Let's leave this open since #180 is already closed. Does adding your api key to a `~/.wakatime.cfg` file in your container and your host machine make it stop prompting?

**Comment by yudi-azvd** [comment]
I don't think so. If I `cat ~/.wakatime.cfg` from inside my container it shows my API key.

**Comment by yudi-azvd** [comment]
Yeah, kinda related. Anyway, the suggestion seems to work. I added the following to `.vscode/devcontainer.json`:

```json
{
    "mounts": [
       "source=${localEnv:HOME}/.wakatime.cfg,target=/home/dev/.wakatime.cfg,type=bind,consistency=delegated"
    ]
}
```

For future readers: mind the path at `target=...`. It should be path that makes sense **in your container**.

@alanhamlett thanks for the help 🤓

**Comment by gandarez** [comment]
We would update the documentation for that scenario.

**Comment by yudi-azvd** [comment]
Actually... after executing `docker compose down` and then `docker compose up -d`, VSCode stills prompts me for the key.

**Comment by nelson6e65** [comment]
I'm not getting request for a new one, but just error instead: 
![imagen](https://github.com/wakatime/vscode-wakatime/assets/9272498/afd0b549-2a16-4d4a-9430-6bad381a268f)


I have it set in my container's `~/wakatime.cfg`, manually:
![imagen](https://github.com/wakatime/vscode-wakatime/assets/9272498/0fb45c14-fb8c-4cba-8855-8fb0e5411ac1)

Also, I added as a global in setting.json:
![imagen](https://github.com/wakatime/vscode-wakatime/assets/9272498/e80d3ad6-e9d3-4f63-9061-6c8af88d3259)

**Comment by alanhamlett** [comment]
Fixed in v25.0.2, it shouldn't prompt anymore when the key is defined in your `settings.json` as `settings.apiKey`.
